doc/administrator_guide.md

```diff
@@ -94,6 +94,28 @@ Select the account key for use with Google Cloud API (which kcidb uses):
 Clone kcidb repository, checkout the revision you want to deploy, and install
 kcidb as described in [README.md](README.md).
 
+### Additional Dependencies
+
+While deploying you may also need to install psql. psql is a command-line tool for interacting with the PostgreSQL database. It is used to manage and manipulate data stored in a PostgreSQL database, as well as to administer and configure the database itself. 
+
+Update the package lists:
+    
+    sudo apt-get update
+
+Install the postgresql-client-common package: 
+
+    apt-get install postgresql-client-common
+
+Install the postgresql-client package, which includes the psql command:
+   
+    apt-get install postgresql-client
```

Comment:
The tool can only be contained in a single package, and if we it needs a dependency, then it should be installed automatically. So I assume only one package needs to be installed. Could you find out which one it is?